image/system_a/data/lang/Polski.json

```diff
@@ -651,11 +651,10 @@
   "app_bellmain_relaxation": "Relaks",
   "app_bellmain_settings": "Ustawienia",
   "app_bellmain_main_window_title": "Mudita Harmony",
-  "app_bell_onboarding_welcome_message": "<text>Mudita Harmony<br/>jest wyłączony</text>",
-  "app_bell_onboarding_info_rotate": "<text weight='regular' size='38'>Obróć, </text><text weight='light' size='38'>aby wybrać</text>",
-  "app_bell_onboarding_info_light_click": "<text weight='regular' size='38'>Kliknij lekko, </text><text weight='light' size='38'>aby kontynuować</text>",
-  "app_bell_onboarding_info_deep_click_warning": "<text weight='light' size='38'>Głęboko </text><text weight='regular' size='38'>wciśnięty</text>",
-  "app_bell_onboarding_info_deep_click_correction": "<text weight='light' size='38'>Bądź bardziej delikatny,<br></br>spróbuj </text><text weight='regular' size='38'>tym razem<br></br></text><text weight='light' size='38'>lekko kliknąć</text>",
+  "app_bell_onboarding_info_rotate": "Obróć, aby wybrać",
+  "app_bell_onboarding_info_light_click": "Kliknij lekko, aby kontynuować",
```

Comment:
same here

Reply:
This text is simply consistent with the previous one. See the explanation above.